src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -58,10 +59,29 @@
  * @since 07.01.2015
  */
 @Slf4j
+@RequiredArgsConstructor
 @Configuration
 @EnableWebSecurity
 public class SecurityConfiguration {
 
+    private final WebUserRepository webUserRepository;
+
+    @PostConstruct
```

Comment:
Not sure if post construct of a spring configuration is the best place for logic.

Instead, what do you think about:

```java
@RequiredArgsConstructor
@Component
public static class AutoCreateAdminAtStartup {
    private final WebUserRepository webUserRepository;

    @EventListener
    public void handleContextStart(ContextStartedEvent cse) {
        if (webUserRepository.hasUserWithAuthority("ADMIN")) {
            return;
        }

        var user = User
            .withUsername(SteveConfiguration.CONFIG.getAuth().getUserName())
            .password(SteveConfiguration.CONFIG.getAuth().getEncodedPassword())
            .disabled(false)
            .authorities("ADMIN")
            .build();

        webUserRepository.createUser(user);
    }
}
```

Reply:
done in https://github.com/steve-community/steve/commit/0992d5cdc3f13ead84e1d09157b8d1bfdd80bfde

however, i did it as 
```
@EventListener
public void afterStart(ContextRefreshedEvent event) {
   ...
}
```
since your suggested method was not being triggered. this is more in line with some other usage in the repo anyways.